orc8r/cloud/go/obsidian/server/server.go

```diff
@@ -101,10 +106,20 @@ func Start() {
 		e.Use(access.Middleware)
 	}
 
-	e, err = reverse_proxy.AddReverseProxyPaths(e)
+	reverseProxyHandler := reverse_proxy.NewReverseProxyHandler()
+	e, err = reverseProxyHandler.AddReverseProxyPaths(e)
 	if err != nil {
 		log.Fatalf("Error adding reverse proxy paths: %s", err)
 	}
+	go func() {
+		for {
+			<-time.After(reverseProxyRefreshPeriod)
+			e, err = reverseProxyHandler.AddReverseProxyPaths(e)
```

Comment:
hm maybe we should inject a service registry querier as an interface dependency to the proxy instead so we can have different impls for docker-compose dev setup and k8s